experiments.json

```diff
@@ -33,5 +33,32 @@
       "min": "0",
       "max": "0"
     }
+  },
+  "content-notifications-12hrs": {
+    "match": {
+      "appId": "^org.mozilla.fennec"
```

Comment:
This wasn't my intention. I updated the patch to be clear and avoid confusion when the code rides the trains.